lab-anthony/index.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
```

Comment:
This is fantastic--I really like that you are breaking up the paths here and passing them into the read function as an argument.